caps/eth.md

```diff
@@ -0,0 +1,223 @@
+# The eth/63 protocol
+
+'eth' is protocol on the [RLPx](../rlpx.md) transport that facilitates exchange of
+Ethereum blockchain information between peers. The current protocol version is **eth/63**.
+See end of document for a list of changes in past protocol versions.
+
+### Protocol Semantics
+
+Upon an active session, a Status message must be sent. Following the reception of the
+peer's Status message, the Ethereum session is active and any other messages may be sent.
+
+All known transactions should be sent following the Status exchange with one or more
+Transactions messages.
+
+Transactions messages should also be sent periodically as the node has new transactions to
```

Comment:
```suggestion
Transaction messages should also be sent periodically as the node has new transactions to
```

Reply:
This refers to the message 'Transactions'